src/client/Microsoft.Identity.Client/IConfidentialClientApplication.cs

```diff
@@ -75,6 +75,12 @@ public partial interface IConfidentialClientApplication : IClientApplicationBase
         /// </remarks>
         AcquireTokenOnBehalfOfParameterBuilder AcquireTokenOnBehalfOf(IEnumerable<string> scopes, UserAssertion userAssertion);
 
+        /// <summary>
+        /// Please use <see cref="IClientApplicationBase.AcquireTokenSilent(IEnumerable{string}, IAccount)"/>.
+        /// </summary>
+        [Obsolete("Use AcquireTokenSilent(scopes, account)", true)]
+        AcquireTokenSilentParameterBuilder AcquireTokenSilent(IEnumerable<string> scopes, string loginHint);
```

Comment:
I think this needs `new` and implementation in CCA class is not needed, similar to 
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/7fff414e166d6e10eb68b44802830749a91841fb/src/client/Microsoft.Identity.Client/IConfidentialClientApplication.cs#L95-L97